appcompat-extension/src/main/java/com/tr4android/support/extension/typeface/TypefaceUtils.java

```diff
@@ -31,31 +31,29 @@
      * Returns true if all letters of the English alphabet of the typefaces are the same.
      */
     @TargetApi(Build.VERSION_CODES.HONEYCOMB_MR1)
-    public static boolean sameAs(Typeface typeface1, Typeface typeface2){
-        // Check if one of the typefaces is null
-        if(typeface1 == null){
+    public static boolean sameAs(Typeface typeface1, Typeface typeface2) {
+        // Handle null as param.
+        if (typeface1 == null) {
             return typeface2 == null;
-        } else if(typeface2 == null){
+        } else if (typeface2 == null) {
             return false; //result of typeface1 == null
         }
 
-        // Check if the letters of the English alphabet of the typefaces are the same
-        Paint paint1 = new Paint();
-        paint1.setTypeface(typeface1);
-        Rect bounds1 = new Rect();
-        paint1.getTextBounds(TEXT, 0, TEXT.length(), bounds1);
-        Bitmap bitmap1 = Bitmap.createBitmap(bounds1.width(), bounds1.height(), Bitmap.Config.ALPHA_8);
-        Canvas canvas1 = new Canvas(bitmap1);
-        canvas1.drawText(TEXT, 0, 0, paint1);
+        // Check if the letters of the English alphabet of the typefaces are the same.
+        Paint paint = new Paint();
+        paint.setTypeface(typeface1);
+        Rect bounds = new Rect();
```

Comment:
@fountaingeyser If the bounds are not equal we could skip the expensive `bitmap1.sameAs(bitmap2)` call since they can't possibly be the same fonts.
Also, could you please make this behavior optional, so developers can disable it if they choose to?